pkg/containerengine/docker.go

```diff
@@ -58,20 +54,24 @@ func newDocker() (ContainerEngine, error) {
 	cmd = exec.Command("systemctl", "is-active", "docker")
 	err = cmd.Run()
 	if err != nil || cmd.ProcessState.ExitCode() != 0 {
-		fmt.Println("docker service not running, starting..")
-		cmd = exec.Command("systemctl", "start", "docker")
-		cmd.Stdin = os.Stdin
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
-		err = cmd.Run()
-		if err != nil {
+		if runtime.GOOS == "linux" {
+			fmt.Println("docker service not running, starting..")
+			cmd = exec.Command("systemctl", "start", "docker")
+			cmd.Stdin = os.Stdin
+			cmd.Stdout = os.Stdout
+			cmd.Stderr = os.Stderr
+			err = cmd.Run()
+			if err != nil {
+				return nil, err
+			}
```

Comment:
Will this require privileged execution in some environments? Might be better to provide an informative error for now.

Reply:
it pops up a dialog on my system and asks for sudo password